src/WaferCircle.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { CSSProperties, FunctionComponent, useMemo } from 'react';
+
+interface WaferCircleProps {
+  diameter: number;
+  size: number;
+}
+
+function getCircleStyle(diameter: number): CSSProperties {
+  return {
+    height: diameter,
+    width: diameter,
+    position: 'relative',
+    borderRadius: '50%',
+    background: '#549ADA',
+  };
+}
+const WaferCircle: FunctionComponent<WaferCircleProps> = ({
+  diameter,
+  size,
+}) => {
+  const circleStyle = useMemo(() => getCircleStyle(diameter), [diameter]);
+  const containerStyle: CSSProperties = {
+    position: 'absolute',
+    width: size,
+    height: size,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: 'white',
+    zIndex: -2,
+  };
+  return (
+    <div style={containerStyle}>
+      <div style={circleStyle} />
+    </div>
```

Comment:
Any reason to not use SVG ?

Reply:
As far as I know [an external package is required](https://react-svgr.com/) for doing this, and I'm not using all the features of SVG

Reply:
I'm men's directly render SVG in React. Not using external images. We do it often and it doesn't require a library other than react-dom